utils/analytics.js

```diff
@@ -1,10 +1,18 @@
 import ReactGA from "react-ga";
 
 export const initGA = () => {
-  ReactGA.initialize([
-    { trackingId: process.env.GA_UA },
-    { trackingId: process.env.GA_UA_CDS }
-  ]);
+  ReactGA.initialize(
+    [
+      {
+        trackingId: process.env.GA_UA,
+        gaOptions: { name: "tracker_1" }
+      },
+      {
+        trackingId: process.env.GA_UA_CDS
+      }
+    ]
+    // , { debug: true }
```

Comment:
leaving this in since it will be useful if we ever need to modify this file again